cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/common/utils.go

```diff
@@ -124,6 +111,17 @@ func NamespacedName(obj ctrlclient.Object) (*types.NamespacedName, error) {
 	}
 }
 
+// "Local" in the sense of no namespace is specified
+func NewLocalOpaqueSecret(name string) *apiv1.Secret {
+	return &apiv1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: name,
```

Comment:
Oh - GenerateName looks useful. I'd seen StringData for secrets before, but not GenerateName for metadata.

Reply:
👍